src/Laravel/LaravelNorthstarClient.php

```diff
@@ -1,25 +1,33 @@
 <?php
 
-namespace DoSomething\Northstar;
+namespace DoSomething\Northstar\Laravel;
 
-use GuzzleHttp\Message\Response;
-use Illuminate\Support\MessageBag;
+use DoSomething\Northstar\NorthstarClient;
 use DoSomething\Northstar\Exceptions\InternalException;
 use DoSomething\Northstar\Exceptions\ValidationException;
 use Symfony\Component\HttpKernel\Exception\HttpException;
 use Illuminate\Contracts\Validation\ValidationException as LaravelValidationException;
+use Illuminate\Support\MessageBag;
 
 class LaravelNorthstarClient extends NorthstarClient
 {
     /**
+     * The class name of the OAuth repository. For Laravel, we default to the included repository
+     * (although that can be overridden with the $config['repository'] option in the constructor).
+     *
+     * @var string
+     */
+    protected $repository = \DoSomething\Northstar\Laravel\LaravelOAuthRepository::class;
+
+    /**
      * Send a Northstar API request, and translates any Northstar exceptions
      * into their built-in Laravel equivalents.
      *
      * @param string $method
      * @param string $path
      * @param array $options
      * @param bool $withAuthorization
-     * @return Response|void
+     * @return \GuzzleHttp\Psr7\Response|void
```

Comment:
That was the namespace for the old Guzzle 5 response class, which I forgot to remove in my last pull request. I didn't import the new class via a `use` statement because it's only used in the docblock here.